Source/WebCore/html/HTMLPreElement.cpp

```diff
@@ -54,8 +54,10 @@ bool HTMLPreElement::hasPresentationalHintsForAttribute(const QualifiedName& nam
 
 void HTMLPreElement::collectPresentationalHintsForAttribute(const QualifiedName& name, const AtomString& value, MutableStyleProperties& style)
 {
-    if (name == wrapAttr)
-        style.setProperty(CSSPropertyWhiteSpace, CSSValuePreWrap);
+    if (name == wrapAttr) {
+        style.setProperty(CSSPropertyWhiteSpaceCollapse, CSSValuePreserve);
+        style.setProperty(CSSPropertyTextWrap, CSSValueWrap);
+    }
     else
```

Comment:
The else goes on the previous line

```suggestion
    } else
```